### DISCUSSION TITLE
Apps over 25 megabytes

### DISCUSSION THREAD

**Discussion by upintheairsheep** [seedvault-app/seedvault#600] [Q&A]
Do apps that have more than 25 megabytes of data back up? Google Drive has an annoying 25 MB limit but I don’t know if it’s on Google Drive’s end or the android backup manager’s end.

**Comment by grote** [comment]
They back up, if you go to experimental settings and activate the unlimited quota option. Otherwise Seedvault has a 50MB limit.
